frontend/src/core/term/components/Term.js

```diff
@@ -48,6 +58,14 @@ export default function Term(props: Props) {
         <header>
             <span className='text'>{ term.text }</span>
             <span className='part-of-speech'>{ term.partOfSpeech }</span>
+            <a
+                href={ `/${locale}/terminology/common/?string=${term.entityId}` }
+                onClick={ navigateToPath }
+                className='translate'
+            >
```

Comment:
Wouldn't this work with a simple `<Link>` tag from `react-router`? I'm guessing I'm the one who built this complicated `navigateToPath` thing, but looking at this code I wonder if it's needed.

Reply:
I don't think that's possible, because we want to check for unsaved changes.